### PR TITLE
Fix return type of getButtonElement()

### PR DESCRIPTION
**Pull request by armno**
## Description

Hi @hodgef ,

The problem I had was that my TSLint complains about the `void` type cannot be tested as a boolean. For example.

screen shot 2018-12-04 at 10 44 23

So in this pull request, I fix return type of [`getButtonElement(name)`](https://franciscohodge.com/projects/simple-keyboard/documentation/#getButtonElement) method. It should not have a return type of `void` (my bad! 😅 ).

From the [source code](https://github.com/hodgef/simple-keyboard/blob/master/src/lib/components/Keyboard.js#L490), it should return an HTML element or a list of HTML elements if there are multiple elements with the same name.

There is another method: [`getInput()`](https://github.com/hodgef/simple-keyboard/blob/master/src/lib/components/Keyboard.js#L312) which currently has a return type of `void` in `index.d.ts` file, which I believe it is also not correct. But I don't know exactly what type should it be as I'm not using this very method in my app. Maybe you can give a hint? 

## Checks

- [x] Tests ( `npm run test -- --coverage` ) Coverage at `./coverage/lcov-report/index.html` should be 100%
- [x] Documentation ( `npm run docs` ) Coverage at `./docs/source.html` should be 100%

### CODE REVIEW

**Comment by hodgef** [comment]
Hey @armno, thanks for reporting this and putting forward a PR! I'll try this out on the [angular test repo](https://github.com/hodgef/angular-simple-keyboard/tree/master/) and merge in a few moments. Thanks again!

**Comment by hodgef** [comment]
Hey @armno, thanks for reporting this and putting forward a PR!

Indeed, the `getButtonElement` method works as you described. It definitely should have the `HTMLElement` type so I'm sorry for not flagging this in my review 😅 The `getInput` method returns the current input string. Basically:

```
let keyboard = new Keyboard();

// then type
> abc

// You get:
console.log(
  keyboard.getInput() // returns "abc"
);
```

Also:

```
let keyboard = new Keyboard({
  inputName: "myInput"
});

// then type
> abc

// You get:
console.log(
  keyboard.getInput("myInput") // returns "abc"
);
```

This functionality basically allows simple-keyboard to work without needing an input element, since it keeps its values in an internal non persistent variable.

Long story short, this method returns a string 😄 

I've made this change to the `index.d.ts` and updated the builds. If you see anything missing, do let me know !

Thank you
